integration/oasis/stationgraph/graph.go

```diff
@@ -1,13 +1,15 @@
 package stationgraph
 
 import (
+	"github.com/Telenav/osrm-backend/integration/oasis/chargingstrategy"
 	"github.com/golang/glog"
 )
 
 type graph struct {
 	nodes       []*node
 	startNodeID nodeID
 	endNodeID   nodeID
+	strategy    chargingstrategy.ChargingStrategyCreator
```

Comment:
Nit: could be `chargingStrategy.Createor`

Reply:
`ChargingStrategyCreator` is a name which is a bit too long, this interface inside of `chargestrategy` package, so no need add `ChargingStrategy`, I agree with that.

I will record this suggestion in #211 and change the name in the future, depending on the behavior of that interface, a better name could be `Creator`/`Charger`